src/cortxfs/cortxfs_ops.c

```diff
@@ -597,6 +597,8 @@ int cfs_rename(struct cfs_fs *cfs_fs, cfs_cred_t *cred,
 			RC_WRAP_LABEL(rc, out, cfs_detach, cfs_fs, cred,
 				      dino_dir, &dino, dname);
 		}
+	} else {
+		ino_to_node_id(dino_dir, &dnode_id);
```

Comment:
Question: We are setting dino_id for both if and else. Should this be in the common code before both if and else constructs? That way dnode_id will always contain a valid value.

Reply:
Both values are different hence if and else. So we cant make it common.

Reply:
Okay.